LICENSE

```diff
@@ -1,8 +1,6 @@
 MIT License
 
-Copyright (c) James A. Fellows Yates, Thiseas C. Lamnidis, Maxime Borry, 
-Maxime Garcia, Aida Andrades Valtueña, Zandra Fagernäs, Stephen Clayton, 
-Judith Neukamm, Alexander Peltzer
+Copyright (c) The nf-core/eager community
```

Comment:
I like that nf-core/eager community

Reply:
Might just not hold from a legal perspective which tends to require a "real person" (or multiple). But I guess for an aDNA pipeline this sounds fine ;-)